client/model.go

```diff
@@ -40,6 +40,73 @@ type ProjectCreatePayload struct {
 	Description string `json:"description"`
 }
 
+type EnvironmentCreate struct {
+	Name                        string         `json:"name"`
+	ProjectId                   string         `json:"projectId"`
+	DeployRequest               *DeployRequest `json:"deployRequest"`
+	WorkspaceName               string         `json:"workspaceName,omitempty"`
+	RequiresApproval            bool           `json:"requiresApproval,omitempty"`
+	ContinuousDeployment        bool           `json:"continuousDeployment,omitempty"`
+	PullRequestPlanDeployments  bool           `json:"pullRequestPlanDeployments,omitempty"`
+	AutoDeployOnPathChangesOnly bool           `json:"autoDeployOnPathChangesOnly,omitempty"`
+	AutoDeployByCustomGlob      bool           `json:"autoDeployByCustomGlob,omitempty"`
+}
+
+type DeployRequest struct {
+	BlueprintId          string                `json:"blueprintId,omitempty"`
+	BlueprintRevision    string                `json:"blueprintRevision,omitempty"`
+	BlueprintRepository  string                `json:"blueprintRepository,omitempty"`
+	ConfigurationChanges *ConfigurationChanges `json:"configurationChanges,omitempty"`
+	TTL                  *TTL                  `json:"ttl,omitempty"`
+	EnvName              string                `json:"envName,omitempty"`
+	UserRequiresApproval bool                  `json:"userRequiresApproval,omitempty"`
+	Targets              string                `json:"targets,omitempty"`
+	PrPlanOnly           bool                  `json:"prPlanOnly,omitempty"`
+	// TODO: not sure about this one
+	CustomEnv0EnvironmentVariables *interface{} `json:"customEnv0EnvironmentVariables,omitempty"`
+	GitUserData                    *GitUserData `json:"gitUserData,omitempty"`
+	TriggerName                    string       `json:"triggerName,omitempty"`
+}
+
+type GitUserData struct {
+	GitUser      string `json:"gitUser,omitempty"`
+	GitAvatarUrl string `json:"gitAvatarUrl,omitempty"`
+	PrNumber     string `json:"prNumber,omitempty"`
+}
+
+type TTL struct {
+	Type  string `json:"type"`
+	Value string `json:"value,omitempty"`
+}
+
+type ConfigurationChanges struct {
+	Name        string                      `json:"name"`
+	Value       string                      `json:"value"`
+	ScopeId     string                      `json:"scopeId,omitempty"`
+	Scope       string                      `json:"scope"`
+	Type        string                      `json:"type"`
+	ToDelete    string                      `json:"toDelete,omitempty"`
+	Schema      *ConfigurationChangesSchema `json:"schema,omitempty"`
+	Description string                      `json:"description,omitempty"`
+	IsSensitive string                      `json:"isSensitive,omitempty"`
+}
+
+type ConfigurationChangesSchema struct {
+	Type string `json:"type,omitempty"`
+	// TODO: not sure this works
```

Comment:
like the one above (this is part of [deployment request](https://developer.env0.com/docs/api/b3A6MTkzNDg1MTk-create-environment))